app/screens/ExploreScreen/ExploreScreen.tsx

```diff
@@ -56,40 +57,39 @@ const useRecommendationSections = (): {
 
   const rawRecs = groupBy("type")(recommendations)
   const recs = Object.keys(rawRecs).reduce<GroupedRecommendations>(
-    (acc, recommendationType) => ({
+    (acc, recommendationTypeId) => ({
       ...acc,
-      [WEBFLOW_MAP.recommendationType[recommendationType]]: rawRecs[recommendationType] ?? [],
+      [WEBFLOW_MAP.recommendationType[recommendationTypeId as RecommendationTypeId]]:
+        rawRecs[recommendationTypeId] ?? [],
     }),
     initialRecs,
   )
   const sortedRecs = customSortObjectKeys(recs, ["Food/Drink", "Unique/to/Portland"])
 
   return {
     isLoading,
-    sections: Object.entries(sortedRecs).map(
-      ([key, value]: [RecommendationType, RawRecommendations[]]) => ({
-        title: sectionTitle(key),
-        renderItem: RenderItem,
-        data: [
-          value.map(
-            (item) =>
-              ({
-                image: item.images.map((image) => ({ uri: image.url })),
-                subtitle: item.name,
-                meta: item.descriptor,
-                body: item.description,
-                leftButton:
-                  item["street-address"] && item["city-state-zip"]
-                    ? {
-                        text: translate("exploreScreen.openInMaps"),
-                        link: `${item["street-address"]},${item["city-state-zip"]}`,
-                      }
-                    : undefined,
-              } as DynamicCarouselItem),
-          ),
-        ],
-      }),
-    ),
+    sections: Object.entries(sortedRecs).map(([key, value]: [string, RawRecommendations[]]) => ({
+      title: sectionTitle(key as RecommendationType),
```

Comment:
@joshuayoes If you know how I can put `RecommendationType` back up where `string` is. This was the least invasive way I saw.

Reply:
Unfortunately, `Object.entries` only lets you set the value type, which is not technically correct since objects can have strings, Symbols, or narrower string unions as keys. People get around this by using `as`, which doesn't feel great, but is the right answer here. https://stackoverflow.com/a/73913412

You could also write a utility function to encapsulate the type cast.

Here are some suggestion comments for how you can fix this using `as`:
https://github.com/infinitered/ChainReactApp2023/pull/212/files/63fe8f1597024406b9827da469882a7d7260660c..cd33ec17f785245f19712ad3c6dfe67b9c4a45b1#r1188900803
https://github.com/infinitered/ChainReactApp2023/pull/212/files/63fe8f1597024406b9827da469882a7d7260660c..cd33ec17f785245f19712ad3c6dfe67b9c4a45b1#r1188899473

Reply:
This feels like it should be a `ts-reset` thing, but apparently they aren't a huge fan since Objects can get mutated 🤷  https://github.com/total-typescript/ts-reset#objectkeysobjectentries